gensim/models/fasttext.py

```diff
@@ -1135,7 +941,7 @@ def save_facebook_model(model, path, encoding="utf-8", lr_update_rate=100, word_
 
 
 class FastTextKeyedVectors(KeyedVectors):
-    def __init__(self, vector_size, min_n, max_n, bucket):
+    def __init__(self, vector_size, min_n, max_n, bucket, count=0, dtype=REAL):
```

Comment:
We should document these new parameters.

Reply:
I'll doc here and in superclass as part of the mapfile work that also touches them.